presto-product-tests/README.md

```diff
@@ -147,6 +147,9 @@ where [profile](#profile) is one of either:
  installation running on a single Docker container and a single node
  installation of kerberized Presto also running on a single Docker container.
  This profile runs Kerberos without impersonation.
+- **singlenode-ldap** - Three single node Docker containers, one running OpenLDAP
+ server, another with SSL/TLS certificates installed on top of a single node Presto
+ installation, and third one with a pseudo-distributed Hadoop installation
```

Comment:
Three single node Docker containers, one running an OpenLDAP
server, one running with SSL/TLS certificates installed on top of a single node Presto
installation, and one with a pseudo-distributed Hadoop installation.